app/src/main/java/org/oppia/android/app/profile/AddProfileActivity.kt

```diff
@@ -26,6 +26,7 @@ class AddProfileActivity : InjectableAppCompatActivity() {
     super.onCreate(savedInstanceState)
     (activityComponent as ActivityComponentImpl).inject(this)
     addProfileFragmentPresenter.handleOnCreate()
+    intent = addProfileFragmentPresenter.getCurrentAppScreenNameIntent()
```

Comment:
We should be adding the value during intent creation, not overriding it here (as that may have unexpected behaviors--we rely on the intent being present in a lot of cases.

Reply:
+1

Reply:
noted, done.